spec/unit/simple_datastream_spec.rb

```diff
@@ -62,5 +62,41 @@
       expect(solr[ActiveFedora::SolrQueryBuilder.solr_name('creation_date', type: :date)]).to eq "2012-01-15"
     end
   end
+  
+  describe "datastream configuration" do
+    before do
+      class FooHistory < ActiveFedora::Base
+        has_metadata :type=>ActiveFedora::SimpleDatastream, :name=>"someData" do |m|
+          m.field "fubar", :string
+        end
+        has_attributes :fubar, datastream: 'someData', multiple: false
+      end
+    end
+    
+    after do
+      Object.send(:remove_const, :FooHistory)
+    end
+
+    let(:foo_id) {
+      f = ActiveFedora::Base.create! do |obj|
+        obj.add_file(%{<?xml version="1.0"?>\n<fields><fubar>test</fubar></fields>}, path:'someData')
+      end
+      client = ActiveFedora.fedora.connection
+      resource = Ldp::Resource::RdfSource.new(client, f.uri)
+      orm = Ldp::Orm.new(resource)
+      orm.graph.delete([orm.resource.subject_uri, ActiveFedora::RDF::Fcrepo::Model.hasModel, nil])
+      orm.graph.insert([orm.resource.subject_uri, ActiveFedora::RDF::Fcrepo::Model.hasModel, 'FooHistory'])
+      orm.save
+      f.reload
+      f.update_index
+      f.id
+    }
+    
+    let(:foo) { FooHistory.find(foo_id) }
+    
+    it "should configure the datastream on load" do
+      expect(foo.fubar).to eq 'test'
+    end
```

Comment:
The above 5 lines could be simplified to:
```
subject { FooHistory.find(foo_id).fubar }
it { is_expected.to eq "test" }
```
I don't have any ideas about the big `let` block, though.